app/src/debug/kotlin/com/simplemobiletools/calendar/BuildVariantApplication.kt

```diff
@@ -0,0 +1,14 @@
+package com.simplemobiletools.calendar
+
+import com.squareup.leakcanary.LeakCanary
+
+open class BuildVariantApplication : BaseApp() {
```

Comment:
This is a slightly more complicated approach than just using the `leakcanary-android-no-op` version for release builds, but it's useful to be able to install some libraries only in debug builds, and not all of them come with appropriate no-op versions.